src/NServiceBus.Core.Tests/NServiceBus.Core.Tests.csproj

```diff
@@ -8,6 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\NServiceBus.AcceptanceTesting\NServiceBus.AcceptanceTesting.csproj" />
```

Comment:
We shouldn't reference the acceptance tests from the unit tests so remove this and switch the unit tests back to using the InMemory persistence for now